{"0.0.1":{"info":{"author":null,"author_email":"\"J. Alexánder Alzate Olaya\" <it@jaalzateolaya.com>","bugtrack_url":null,"classifiers":["Development Status :: 4 - Beta","Intended Audience :: Developers","License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3","Topic :: Software Development :: Libraries :: Python Modules"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":"ETL, Data Synchronization","license":"MIT License  Copyright 2024 J. Alexánder Alzate Olaya  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the “Software”), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.  THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. ","maintainer":null,"maintainer_email":null,"name":"pyDataBridge","package_url":"https://pypi.org/project/pyDataBridge/","platform":null,"project_url":"https://pypi.org/project/pyDataBridge/","project_urls":{"Bug Tracker":"https://gitlab.com/jaalzateolaya/pydatabridge/-/issues","Homepage":"https://gitlab.com/jaalzateolaya/pydatabridge"},"provides_extra":null,"release_url":"https://pypi.org/project/pyDataBridge/0.0.1/","requires_dist":["dependency-injector>=4.41","pypika-git"],"requires_python":">=3.7","summary":"A data synchronization framework for python","version":"0.0.1","yanked":false,"yanked_reason":null},"last_serial":23207221,"urls":[{"comment_text":"","digests":{"blake2b_256":"f94741a04abb2c524e9220cd9b198cc6dc97f60ea51829c6c6ae3b77e97588f7","md5":"ccf92c6ada64668050be092f6805637f","sha256":"8922a487a067146cfbc4d960421e999ab3ef23136ba0c57726ef2ce147b48957"},"downloads":-1,"filename":"pyDataBridge-0.0.1-py3-none-any.whl","has_sig":false,"md5_digest":"ccf92c6ada64668050be092f6805637f","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.7","size":20430,"upload_time":"2024-05-14T20:19:44","upload_time_iso_8601":"2024-05-14T20:19:44.781020Z","url":"https://files.pythonhosted.org/packages/f9/47/41a04abb2c524e9220cd9b198cc6dc97f60ea51829c6c6ae3b77e97588f7/pyDataBridge-0.0.1-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}